### PR TITLE
Camel spring boot examples for apm opentrace

### PR DESCRIPTION
**Pull request by EUinUkraine**
This pull request contains example for configuring https://camel.apache.org/components/latest/others/opentracing.html to work with ElasticCo Aplication Perfomance Manger Server - https://www.elastic.co/guide/en/apm/get-started/current/install-and-run.html

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: camel-example-spring-boot-apm-opentracing/pom.xml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+            <artifactId>camel-http-starter</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+
+        <!-- CDI API -->
+        <dependency>
```
Comment (davsclaus) [inline camel-example-spring-boot-apm-opentracing/pom.xml:95]: I dont think spring boot uses CDI annotations

File: camel-example-spring-boot-apm-opentracing/src/main/java/sample/camel/Service1Application.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+ * <p/>
+ * Notice we use the `@CamelOpenTracing` annotation to enable Camel with OpenTracing.
+ */
+@SpringBootApplication
+@CamelOpenTracing
+public class Service1Application {
+    /**
+     * A main method to start this application.
+     */
+
```
Comment (davsclaus) [inline camel-example-spring-boot-apm-opentracing/src/main/java/sample/camel/Service1Application.java:35]: Remove empty line

File: camel-example-spring-boot-apm-opentracing/src/main/java/sample/camel/CamelConfig.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
+package sample.camel;
```
Comment (davsclaus) [inline camel-example-spring-boot-apm-opentracing/src/main/java/sample/camel/CamelConfig.java:1]: Add license header

File: camel-example-spring-boot-apm-opentracing/pom.xml [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-dependencies</artifactId>
+                <version>${spring.boot-version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.camel.springboot</groupId>
+                <artifactId>camel-spring-boot-dependencies</artifactId>
```
Comment (davsclaus) [inline camel-example-spring-boot-apm-opentracing/pom.xml:54]: there is a new camel-spring-boot-bom that is maybe better to use

Comment (EUinUkraine) [reply camel-example-spring-boot-apm-opentracing/pom.xml:54]: Can you please clarify

Comment (davsclaus) [reply camel-example-spring-boot-apm-opentracing/pom.xml:54]: https://camel.apache.org/camel-spring-boot/latest/#_camel_spring_boot_bom_vs_camel_spring_boot_dependencies_bom

File: camel-example-spring-boot-apm-opentracing/src/main/java/sample/camel/CamelConfig.java (hunk 3 above)
Comment (EUinUkraine) [reply camel-example-spring-boot-apm-opentracing/src/main/java/sample/camel/CamelConfig.java:17]: Done

File: camel-example-spring-boot-apm-opentracing/pom.xml (hunk 4 above)
Comment (EUinUkraine) [reply camel-example-spring-boot-apm-opentracing/pom.xml:54]: https://repo.maven.apache.org/maven2/org/apache/camel/springboot/camel-spring-boot-dependencies/ 
3.5.0 is latest in maven

Comment (davsclaus) [reply camel-example-spring-boot-apm-opentracing/pom.xml:54]: camel-spring-boot-bom

**Pull request merged**